relevanceai/http_client.py

```diff
@@ -125,4 +125,42 @@ def make_search_suggestion(self):
 
     def check_auth(self):
         return self.admin._ping()
+
+    def kmeans_cluster(
```

Comment:
I think there Is a way to just change the alias like so:

```
kmeans_cluster = vector_tools.cluster.kmeans_cluster
```

I'll have a think